vcnc-server/vcnc-sampler/app.js

```diff
@@ -88,7 +88,7 @@ function requestHandler(request, response) {
 }
 
 const server = http.createServer(requestHandler);
-server.setTimeout(5000, () => {
+server.setTimeout(5000, () => {  // REVIEW: arguments are backwards; what is the purpose?
```

Comment:
This is server timeout function and it has this syntax. It set the number of milliseconds of inactivity before a socket is presumed to have timed out